cmd/substreams/utils.go

```diff
@@ -54,3 +54,5 @@ func readStopBlockFlag(cmd *cobra.Command, startBlock int64, flagName string, wi
 
 	return endBlock, nil
 }
+
+var networkDefaultEndpointDeprecationWarning string = "DEPRECATION WARNING: default endpoint is deprecated. Please specify a network endpoint using the --substreams-endpoint flag (-e) or by specifying an environment variable in the form SUBSTREAMS_ENDPOINTS_CONFIG_{network} where 'network' is the name of the network for this Substreams"
```

Comment:
```suggestion
var networkDefaultEndpointDeprecationWarning string = "DEPRECATION WARNING: Default endpoint is deprecated. Define a 'network' field in your Substreams manifest in which case the endpoint will be inferred for you. You can also manually specify the endpoint to use by using '--substreams-endpoint <endpoint>' flag ('-e <endpoint>') or by specifying an environment variable in the form SUBSTREAMS_ENDPOINTS_CONFIG_{network} where 'network' is the name of the network defined in your Substreams manifest"
```

Reply:
We should also think about linking the list of network somewhere (@enoldev for info about the upcoming feature).